src/index.js

```diff
@@ -1,8 +1,52 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import './stylesheets/index.css';
+import registerServiceWorker from './util/registerServiceWorker';
+import { BrowserRouter } from 'react-router-dom'
+import emoji from "./images/emoji.svg";
+import Link from "react-router-dom/es/Link";
 
-ReactDOM.render(<App />, document.getElementById('title'));
+// this component will be rendered by our <___Router>
+const App = () => (
```

Comment:
App, Main, and Header should be in directories of their own as they grow larger.

ex:  
```
Pages/
   App/
     App.jsx
     App.css
  Main/
    Main.jsx
    Main.css
components/
  Header/
     Header.jsx
     Header.css
```

Reply:
I'm not sure I understand why Main is a page and Header is a component